packages/kit/src/module/compatibility.ts

```diff
@@ -0,0 +1,53 @@
+import satisfies from 'semver/functions/satisfies.js' // npm/node-semver#381
+import type { Nuxt, NuxtModule } from '@nuxt/schema'
+import { useNuxt } from '../context'
+import { normaliseSemanticVersion } from '../compatibility'
+import { loadNuxtModuleInstance } from './install'
+
+/**
+ * CHeck if a Nuxt module is installed by name.
+ *
+ * This will check both the installed modules and the modules to be installed.
```

Comment:
```suggestion
 * This will check both the installed modules and the modules to be installed. Note
 * that it cannot detect if a module is _going to be_ installed programmatically by another module.
```